libraries/botbuilder-core/src/transcriptLogger.ts

```diff
@@ -47,8 +47,24 @@ export class TranscriptLoggerMiddleware implements Middleware {
 
         // hook up onSend pipeline
         context.onSendActivities(async (ctx: TurnContext, activities: Partial<Activity>[], next2: () => Promise<ResourceResponse[]>) => {
-            // run full pipeline
-            const responses: ResourceResponse[] = await next2();
+            // Run full pipeline and create a pointer to the return value from the pipeline.
+            let nextValue: ResourceResponse[] = await next2();
```

Comment:
Shouldn't these be consts?

Reply:
nextValue should definitely be a const, that was a mistake on my part